Finish transition to Svelte

maxlath opened the pull request:
* Rewrites the last views in Svelte: `notifications_layout` (+ children), `confirmation_modal`, and finally (:tada:) `app_layout` (+ children)
* Replaces uses of handlebars `escapeExpression` by [`escape-html`](https://www.npmjs.com/package/escape-html), to be able to remove `handlebars` as a dependency
* Purges obsolete code :fire: 

This PR touches many files, and as we don't have automated tests for the client, manually testing all around the app is welcome

Remaining traces of Backbone/Marionette will be removed in #539

jum-s commented:
When I click on notifications in the top bar, a list of notifications is displayed, then when I click on a notification `href` seem to be missing:

 ```
	 statusCode: 500,
   context: { message: 'URL constructor:  is not a valid URL.' },
   referer: 'http://localhost:3005/notifications',
 ```

jum-s commented:
When opening "Invite friends by email" modal, and I enter a invalid email, an error flash is displayed, then when I enter a valid email, the error flash should disappear (should insert `flash === null`)

jum-s commented:
When I load the `/settings/notifications` page with  `Receive email notifications` disabled, then i click on the checkbox, it should display the other checkboxes, instead it only show the same unchecked checkbox (`Receive email notifications`)

jum-s commented:
When i click on "send request" from a item modal, then i validate the message to send, I get redirected to `/transactions`, but not transactions appears in the ongoing category `Nothing in this category`, if I reload the transaction appear

jum-s commented:
When I click to archive a transaction, I can read the message `you confirmed that you recovered the book`, then the transaction should be moved to 'archived' category (but its still currently in ongoing)

maxlath commented:
> When I click on notifications in the top bar, a list of notifications is displayed, then when I click on a notification `href` seem to be missing:
> 
> ```
>     statusCode: 500,
>   context: { message: 'URL constructor:  is not a valid URL.' },
>   referer: 'http://localhost:3005/notifications',
> ```
Fixed in 2da2cbd4

> When opening "Invite friends by email" modal, and I enter a invalid email, an error flash is displayed, then when I enter a valid email, the error flash should disappear (should insert `flash === null`)

Fixed in 107753610

> When I load the `/settings/notifications` page with `Receive email notifications` disabled, then i click on the checkbox, it should display the other checkboxes, instead it only show the same unchecked checkbox (`Receive email notifications`)

Fixed in 5c5b49897

> When i click on "send request" from a item modal, then i validate the message to send, I get redirected to `/transactions`, but not transactions appears in the ongoing category `Nothing in this category`, if I reload the transaction appear

Fixed in e8266ee74

> When I click to archive a transaction, I can read the message `you confirmed that you recovered the book`, then the transaction should be moved to 'archived' category (but its still currently in ongoing)

Fixed in 063141406
